test/update_test.rb

```diff
@@ -157,6 +163,8 @@ def default_SetDesktopPattern_stubs
     let(:os_backup_release_content) { File.new("#{DATA_DIR}/etc/tw-os-release").read }
 
     before do
+      allow(Yast::Update.log).to receive(:info).and_call_original
```

Comment:
Using `allow` with just `.and_call_original` is useless, you can simply remove this line.

Reply:
I disagree in this case. I need to `allow` because I am using an `expect` later. I mean, this is allowing to test an specific call to `log#info` no matter the content of previous calls. Did you understand what I mean?